### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by 153957** [leukeleu/wagtail-pg-search-backend#3]
- Configuration options
- Setup guide
- Explain how to add to existing projects
- Limitations
- Documentation in wagtail

**Comment by FlipperPA** [comment]
This looks fantastic for smaller sites like one I'm working on where a full-blown elastic search backend would be overkill. Does it respect `search_fields` just like the standard Wagtail search? If someone starts a skeleton for documentation I'll be happy to contribute as I toy around with it on a side project.

**Comment by 153957** [comment]
Yes, use the same method of adding search_fields in your models as documented in the wagtail documentation:
http://docs.wagtail.io/en/v1.9/topics/search/indexing.html#indexing-extra-fields
